src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -62,8 +62,8 @@ function initStore(queryStore: QueryStore) {
     // ultimate we will phase this out and this information will be stored in router etc.
     //const qSession:any = (window as any).QuerySession;
     var samplesSpecification:any = [];
-    if (serverVars.caseSetProperties.case_set_id === "all") {
-        // "all" means all cases in the queried stud(y/ies) - not an actual case set that could be queried
+    if (["-1", "0", "1", "2", "all"].indexOf(serverVars.caseSetProperties.case_set_id) > -1) {
```

Comment:
you can now use [].includes instead of indexOf.   You don't need to make this change though.